### ISSUE TITLE
loss since 2nd epoch becomes huge and unchanged !!!

### ISSUE THREAD

**Issue by duwangthefirst** [xingyizhou/CenterNet#928]
Recently I'm re-organizing the code and train the detector on my own dataset.
The problem is that loss since 2nd epoch becomes huge and unchanged
To solve the problem, I changed this:
```
loss.backward()
optimizer.step()
```
to:
```
loss.backward()
optimizer.step()
optimizer.zero_grad()
```

**Comment by tbbjymm21** [comment]
Hi，has your problem been solved? How to solve it?
